lib/core/ui/dashboard/home/home_appbar.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:live_score/core/theme/app_theme.dart';
+
+class DashboardAppBar extends AppBar {
+  DashboardAppBar({super.key})
+      : super(
+          forceMaterialTransparency: true,
+          leading: Stack(
+            children: [
+              const Positioned(
+                bottom: 18,
+                left: 22,
+                child: Text(
+                  'LiveScore',
```

Comment:
Please, move string to localisations

Reply:
The dashboard app bar is an extension of appbar not an actuall widget, so I'm not sure how should I call context here for localization purposes.